apps/mobile/package.json

```diff
@@ -9,7 +9,7 @@
     "android:release": "expo run:android --variant release",
     "build": "pnpm lingui",
     "check:cocoapods": "sh scripts/enforce-cocoapods-version.sh",
-    "eas-build-post-install": "cd ../.. && pnpm install && pnpm build",
+    "eas-build-post-install": "cd ../.. && pnpm install && pnpm build:mobile",
```

Comment:
@camerow @edgarkhanzadian I got the mobile build working but it's unclear if I need to run a full `pnpm build` here or if `pnpm build:mobile` will do? 

It seems like we are building all packages via `build` which includes web + extension. Maybe I need to be granular here and just build mobile dependancies?

Reply:
I checked this out and it seems that `pnpm build:mobile` is fine here as in `turbo.json`, the build task has `"dependsOn": ["^build"], meaning for any package you build, Turbo also runs build for its dependency graph.

I've confirmed this manually running and checking the command